modules/osdk-common-prereqs.adoc

```diff
@@ -26,9 +26,9 @@ ifdef::golang[]
 endif::[]
 ifdef::ansible[]
 - link:https://docs.ansible.com/ansible/latest/index.html[Ansible] version v2.9.0+
-- link:https://ansible-runner.readthedocs.io/en/latest/install.html[Ansible Runner] version v1.1.0+
+- link:https://ansible-runner.readthedocs.io/en/latest/install.html[Ansible Runner] version v2.0.2+
 - link:https://github.com/ansible/ansible-runner-http[Ansible Runner HTTP Event Emitter plug-in] version v1.0.0+
-- link:https://pypi.org/project/openshift/[OpenShift Python client] version v0.11.2+
+- link:https://pypi.org/project/openshift/[OpenShift Python client] version v0.12.0+
```

Comment:
This links to version 0.13.1, or the latest page. Is that ok with you?

Reply:
Good catch. It is what upstream links to in their prereqs, so as long as the latest is more than 0.12.0 we are good.